md/datetimes-management-tutorial.md

```diff
@@ -0,0 +1,163 @@
+# How to work with dates and times in forms and BDM
+
+::: warning
+This tutorial is for versions 7.5.0 and newer.  
+:::
+
+For use cases involving dates and times Bonita BPM comes with 3 types of data which you can manipulate through the processes, APIs and forms/pages.  
+This page gives a step by step procedure for creating a process with forms allowing to choose a date (with or without a time) and display it in another step leveraging each of these data types.  
+
+## Date only
+
+The type **DATE ONLY** is for manipulating a date which need no time information. E.g., a birth date.  
+The storage format for this type is an ISO 8601 formatted Character String (e.g. "2017-01-13").  
+
+### Define your business data
+
+* In the studio, start by opening the business data model editor (Menu "Business Data Model" > "Manage...")
+* Create an object with an attribute of type DATE ONLY
+* Create a new process
+* Then, in the process data tab, create a new business variable of the type of your business object
+
+### Define the contract and task operations or business variable initialization script
+
+* In the process/task execution tab > contract section, define the contract inputs using the Add from data... button
+* Select the business variable you created previously
+* Follow the steps to generate a contract with a COMPLEX input containing a DATE ONLY attribute (keep the option to generate the variable initialization script or the operations (for a task) automatically)
+
+### Generate the form
+
+* Generate the UI designer form by clicking on edit in the form section of the process/task execution tab
+* Since you have a DATE ONLY attribute in the contract inputs, a date picker widget is automatically added to the form  
+There is nothing else to do.
+
+::: info
+The date picker widget output is an ISO 8601 formatted String (e.g. "2017-01-13T00:00:00.000Z"). The time information is always set to midnight UTC and it is dropped at contract level when received by the process instantiation / task execution REST API and converted to a `java.time.LocalTime`.
+:::
+
+### Retrieve the date from the database and display it
+
+In another step form (or in any other page) you may want to retrieve the date and display it.  
+In order to do that:  
+* Create a new task after the existing one
+* Generate its form by clicking on edit in the form section of the task execution tab
+* In the UI Designer, you need first to define a new variable of type external API to retrieve the object containing the date attribute using the context:
```

Comment:
Now in the UI Designer, define a variable to retrieve the object containing the date attribute, using the **context**:
1. In the **Variables** panel at the bottom of the editor, click on **Create a new variable** button
2. Enter the **Name** _employee_
3. Select the **Type** _External API_ 
4. Enter the following **Value**: